feature.txt

```diff
@@ -0,0 +1 @@
+Hello world from idea
```

Comment:
It should have been empty at the first commit.

Reply:
Thanks